src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,175 @@
+
+- all permissions to `apps` and `charts` from the `application.giantswarm.io` API group.
+- only `get`, `list`, and `watch` permission to `appcatalogs` and `appcatalogentries` from the `application.giantswarm.io` API group.
+- only `get`, `list`, and `watch` permission for clusters.
+
+The single letter icons in the **Verbs** column each represent one of the seven standard verbs: `get`, `watch`, `list`, `create`, `update`, `patch`, and `delete`. The last items, showing an ellipsis `⋯`, stands for additional, arbitrary verbs. Hovering over these icons with your pointer will reveal additional information in a tooltip.
+
+The same permissions, shown as a partial YAML representation of the [Role](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole) definition, could look like this:
+
+```yaml
+rules:
+- apiGroups:
+  - application.giantswarm.io
+  resources:
+  - apps
+  - charts
+  verbs:
+  - '*'
+- apiGroups:
+  - application.giantswarm.io
+  resources:
+  - appcatalogs
+  - appcatalogentries
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - cluster.x-k8s.io
+  resources:
+  - clusters
+  verbs:
+  - get
+  - list
+  - watch
+```
+
+## Finding roles {#finding-roles}
+
+You can filter the list of roles and cluster roles based on a search term. This term is matched with the role name, all subject names, and all permission details. Examples for possible searches are:
+
+- Show all roles with `read` in the name.
+- Show all roles where `someone@example.com` appears as a subject.
+- Show all roles explicitly granting permission to `events`.
+
+## Granting access {#granting-access}
+
+Granting access to an organization's resources means, in Kubernetes RBAC terms: you assign a role or cluster role to the selected organization's namespace, by creating role bindings in this namespace.
+
+In order to assign a role to a user, group, or service account, follow these steps:
+
+### 1. Select a role
+
+Find and select the role or cluster role to assign within the list on the left hand side. Select the role that defines the permissions you want to grant.
+
+For example, the `cluster-admin` ClusterRole, when assigned here, will grant all permissions within this organization's namespace. The `read-all` ClusterRole instead gives read-only access to all resources in this namespace.
+
+### 2. Double-check permissions
+
+By inspecting the _Permissions_ tab, double check that the role grants only the permissions you are expecting.
+
+### 3. Add subjects
+
+On the _Subjects_ tab, add the group(s) or user(s) you want to assign to the selected role.
+
+When adding a **group** or a **user**, you have to enter the exact user or group identifier as your identity provider presents them. This includes upper and lowercase spelling. User identifiers may or may not be email addresses, according to your set-up.
+
+**Note:** Your Giant Swarm account engineer (AE) is happy to assist in case there are uncertainties about the correct spelling.
```

Comment:
I'd switch 'spelling' with 'format', since I don't think the actual spelling is the problem here. I'm under the impression that it's more about case usage, etc.

Reply:
Will change